apistructs/container.go

```diff
@@ -150,6 +150,7 @@ type Pod struct {
 	IPAddress     string         `json:"ipAddress"`
 	Host          string         `json:"host"`
 	Phase         string         `json:"phase"`
+	IsPodHealthy  bool           `json:"isPodHealthy"`
```

Comment:
Phase and IsPodHealthy can be combined, do not directly reveal the phase of K8S.